File: AuthenticationServices/src/main/java/com/artauction/authenticationservices/Dtos/Authentication/AuthenticationDto.java
package com.artauction.authenticationservices.Dtos.Authentication;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
public class AuthenticationDto {
    // User data
    private int Id;
    private String name;
    private String email;
    private String role;
    private Integer adminId;        // Integer (nullable) instead of int?

    // Authentication data
    private String token;
    private String refreshToken;
    private LocalDateTime expiresOn; // DateTime → LocalDateTime
    private String message;
    private boolean isAuthenticated;
}

File: Project/Backend/src/ArtAuction.Application/DTOs/Authentication/AuthenticationDto.cs
namespace ArtAuction.Application.DTOs.Authentication;

public class AuthenticationDto
{
    // User data
    public int Id { get; set; }
    public string Name { get; set; }
    public string Email { get; set; }
    public string Role { get; set; }
    public int? adminId { get; set; }
    
    // Authentication data
    public string Token { get; set; }
    public string RefreshToken { get; set; }
    public DateTime ExpiresOn { get; set; }
    public string Message { get; set; }
    public bool IsAuthenticated { get; set; }
}